_posts/2021-03-05-status-update.md

```diff
@@ -0,0 +1,176 @@
+---
+layout: post
+title: "2021-03-05: IAC Team Status Update"
+author: sheenaajay
+categories:
+  - team
+  - status
+tags:
+- community
+- releases
+- cloud-ci
+- puppet5-support-drop
+- major-releases
+---
+
+## Community Contributions
+
+We'd like to thank the following people in the Puppet Community for their contributions over this past week:
+
+- [`puppetlabs-acl#231`][puppetlabs-acl-pr-231]: "(maint) Update limitations in README.md", thanks to [luchihoratiu][luchihoratiu]
+- [`puppetlabs-docker#725`][puppetlabs-docker-pr-725]: "Make it possible to use pod's network", thanks to [seriv][seriv]
+- [`puppetlabs-kubernetes#494`][puppetlabs-kubernetes-pr-494]: "Added information about Hiera YAML Lookup; installing a updated version.", thanks to [bitvijays][bitvijays]
+- [`puppetlabs-kubernetes#488`][puppetlabs-kubernetes-pr-488]: "Add note about the kubernetes version format", thanks to [danifr][danifr]
+- [`puppetlabs-postgresql#1239`][puppetlabs-postgresql-pr-1239]: "Add DNF module management", thanks to [ekohl][ekohl]
+- [`puppetlabs-reboot#280`][puppetlabs-reboot-pr-280]: "(MODULES-10955) More robust handling of reboot-task output", thanks to [fetzerms][fetzerms]
+- [`pdk#954`][pdk-pr-954]: "Docs: Install on macOS with brew", thanks to [rwaffen][rwaffen]
+- [`pdk-docker#27`][pdk-docker-pr-27]: "(PF-2333) Remove Anubis content from pdk-docker repo", thanks to [nkanderson][nkanderson]
+- [`pdk-docker#26`][pdk-docker-pr-26]: "(PF-2333) Remove Anubis image from workflow", thanks to [nkanderson][nkanderson]
+- [`pdk-templates#411`][pdk-templates-pr-411]: "Add Puppet 7 tests to .gitlab-ci.yml", thanks to [silug][silug]
+- [`pdk-vanagon#277`][pdk-vanagon-pr-277]: "(MAINT) Bump `packaging` dep to at least 0.99.76 to fix MSI signing", thanks to [scotje][scotje]
+- [`pdk-vanagon#276`][pdk-vanagon-pr-276]: "(MAINT) Refactor `puppet-forge-api` component into `puppet-versions`", thanks to [scotje][scotje]
+
+Check [all the ways to reach us]({% link _posts/2021-01-20-reaching-out.md %}) if you want to directly contact us about anything module related.
+
+## Supported Module Major Releases
+
+The whole team has been super busy this week working on prepping our modules for major releases.
+We have done **34+** major releases on our supported modules with only a few outstanding.
+The release includes the removal of Puppet 5 support.
+If you are running Puppet 5 please note it is no longer supported and we advise you to upgrade to Puppet 7 as soon as you can.
+
```

Comment:
Maybe also mention the removal of RHEL family 5 and SLES 11 support?

Reply:
Perfect @david22swan . Thank you